operators/src/processing/raster_vector_join/mod.rs

```diff
@@ -17,30 +17,53 @@ use async_trait::async_trait;
 use futures::future::join_all;
 use geoengine_datatypes::collections::VectorDataType;
 use geoengine_datatypes::primitives::FeatureDataType;
-use geoengine_datatypes::raster::RasterDataType;
+use geoengine_datatypes::raster::{Pixel, RasterDataType};
 use serde::{Deserialize, Serialize};
 use snafu::ensure;
 
+use self::aggregator::{
+    Aggregator, FirstValueFloatAggregator, FirstValueIntAggregator, MeanValueAggregator,
+    TypedAggregator,
+};
+
 /// An operator that attaches raster values to vector data
 pub type RasterVectorJoin = Operator<RasterVectorJoinParams, SingleVectorMultipleRasterSources>;
 
 const MAX_NUMBER_OF_RASTER_INPUTS: usize = 8;
 
 /// The parameter spec for `RasterVectorJoin`
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct RasterVectorJoinParams {
     /// Each name reflects the output column of the join result.
     /// For each raster input, one name must be defined.
     pub names: Vec<String>,
 
-    /// Specifies which method is used for aggregating values
-    pub aggregation: AggregationMethod,
+    /// Specifies which method is used for aggregating values for a feature
+    pub feature_aggregation: FeatureAggregationMethod,
+
+    /// Specifies which method is used for aggregating values over time
+    pub temporal_aggregation: TemporalAggregationMethod,
+}
+
+/// How to aggreagate the values for the geometries inside a feature
+/// e.g. add all the raster values corresponding to the individual
+/// points inside a `MultiPoint` feature. Choosing None leads to an
```

Comment:
there is no none

Reply:
b3ed11c817df2a474264b18f148eedec8b11e407